src/Components/Tasks/Tasks.jsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+import ModalTask from "./ModalTask/ModalTask";
+import { useState } from "react";
+import styles from "./tasks.module.css";
+import Modal from "react-overlays/Modal";
+
+function Tasks({ tasks, changeState, deleteTask, editTask }) {
+  const [showModal, setShowModal] = useState(false);
+  const [flagEditing, setFlagEditing] = useState(false);
```

Comment:
I use these for inline editing, when I click on the task to edit, input will appear instead of a task, so there is a value handle for input change, id to handle id that will change, without id, all tasks will turn to inputs when I choose an editing task, flagEdditting use to change between input and default appearance as a paragraph for displaying tasks.